spec/message.abnf

```diff
@@ -61,13 +61,15 @@ reserved-statement = reserved-keyword [s reserved-body] 1*([s] expression)
 reserved-keyword   = "." name
 
 ; Reserve additional sigils for use by future versions of this specification.
-reserved-annotation       = reserved-annotation-start reserved-body
+reserved-annotation       = reserved-annotation-start [[s] reserved-body]
 reserved-annotation-start = "!" / "%" / "*" / "+" / "<" / ">" / "?" / "~"
 
 ; Reserve sigils for private-use by implementations.
-private-use-annotation    = private-start reserved-body
+private-use-annotation    = private-start [[s] reserved-body]
 private-start             = "^" / "&"
-reserved-body             = *([s] 1*(reserved-char / reserved-escape / quoted))
+reserved-body             = reserved-body-start *reserved-body-part
+reserved-body-start       = reserved-char / reserved-escape / quoted
+reserved-body-part        = [s] 1*(reserved-char / reserved-escape / quoted)
```

Comment:
This can be simplified:

```suggestion
reserved-body             = reserved-body-part *([s] reserved-body-part)
reserved-body-part        = reserved-char / reserved-escape / quoted
```

Reply:
Thanks for the suggestion. I like it, and I'll apply it if there is consensus. (I had introduced the `reserved-body-start` nonterminal following @aphillips comment https://github.com/unicode-org/message-format-wg/pull/730#discussion_r1526378593 .)

Reply:
Let's apply this here and below.